lib/getTwitterAvatarByHandle.ts

```diff
@@ -1,14 +1,17 @@
-import TwitterApi from "twitter-api-v2"
```

Comment:
do we still need `twitter-api-v2` in `package.json`?
- if it's not being used anywhere else, can we remove the package?
- if it **is** being used elsewhere, ignore this comment.